Makefile

```diff
@@ -609,7 +609,7 @@ minikube:
 # make minio-secrets
 
 #<<<
-minio-secrets:
+minio-secrets: ssl-cert
```

Comment:
since `ssl-cert` is already run with `init-docker`, do we need to run it again here?

Reply:
probably not...I'll take that out